tests/testdata/mocked_service.py

```diff
@@ -11,7 +11,10 @@ async def some_endpoint(
     item: Dict[str, str],
     limit: Optional[int] = None,
     skip: Optional[int] = None,
+    x_correlation_id: Annotated[list[str] | None, fastapi.Header()] = None,
```

Comment:
NB: strange to use a list for an id. Is there something special with supporting a `list[str]` compared to plain `str` ?

Reply:
Copied from FastAPI docs.
This is probably because nothing forbids you to add several time the same header key...

Reply:
Au final, en lisant mieux la doc, j'ai pu m'apercevoir qu'on était pas obligé de passer par une liste